File: app/src/main/java/br/com/brqtest/model/Endereco.java
package br.com.brqtest.model;

import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;

@DatabaseTable(tableName = "endereco")
public class Endereco extends EntidadeDominio {

    @DatabaseField(generatedId = true)
    private Integer id;
    @DatabaseField
    private String logradouro;
    @DatabaseField
    private String numero;
    @DatabaseField
    private String complemento;
    @DatabaseField
    private String bairro;
    @DatabaseField
    private String cidade;
    private String localidade;
    @DatabaseField
    private String uf;
    @DatabaseField
    private String cep;

    public Endereco(Integer id){
        setId(id);
    }

    public Endereco(){

    }

    public String getLogradouro() {
        return logradouro;
    }

    public void setLogradouro(String logradouro) {
        this.logradouro = logradouro;
    }

    public String getNumero() {
        return numero;
    }

    public void setNumero(String numero) {
        this.numero = numero;
    }

    public String getComplemento() {
        return complemento;
    }

    public void setComplemento(String complemento) {
        this.complemento = complemento;
    }

    public String getBairro() {
        return bairro;
    }

    public void setBairro(String bairro) {
        this.bairro = bairro;
    }

    public String getCidade() {
        return cidade;
    }

    public void setCidade(String cidade) {
        this.cidade = cidade;
    }

    public String getUf() {
        return uf;
    }

    public void setUf(String uf) {
        this.uf = uf;
    }

    public String getCep() {
        return cep;
    }

    public void setCep(String cep) {
        this.cep = cep;
    }

    @Override
    public String toString() {
        return "Endereco{" +
                "logradouro='" + logradouro + '\'' +
                ", numero='" + numero + '\'' +
                ", complemento='" + complemento + '\'' +
                ", bairro='" + bairro + '\'' +
                ", cidade='" + cidade + '\'' +
                ", uf='" + uf + '\'' +
                ", cep='" + cep + '\'' +
                '}';
    }

    public String getLocalidade() {
        return localidade;
    }

    public void setLocalidade(String localidade) {
        this.localidade = localidade;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }
}

File: app/src/main/java/br/com/brqtest/modelkotlin/Endereco.kt
package br.com.brqtest.modelkotlin

class Endereco:EntidadeDominio() {

    private var logradouro: String? = null
    private var numero: String? = null
    private var complemento: String? = null
    private var bairro: String? = null
    private var cidade: String? = null
    private var uf: String? = null
    private var cep: String? = null

    fun Endereco(id: Int?) {
        setId(id)
    }

    fun Endereco() {

    }

    fun getLogradouro(): String? {
        return logradouro
    }

    fun setLogradouro(logradouro: String) {
        this.logradouro = logradouro
    }

    fun getNumero(): String? {
        return numero
    }

    fun setNumero(numero: String) {
        this.numero = numero
    }

    fun getComplemento(): String? {
        return complemento
    }

    fun setComplemento(complemento: String) {
        this.complemento = complemento
    }

    fun getBairro(): String? {
        return bairro
    }

    fun setBairro(bairro: String) {
        this.bairro = bairro
    }

    fun getCidade(): String? {
        return cidade
    }

    fun setCidade(cidade: String) {
        this.cidade = cidade
    }

    fun getUf(): String? {
        return uf
    }

    fun setUf(uf: String) {
        this.uf = uf
    }

    fun getCep(): String? {
        return cep
    }

    fun setCep(cep: String) {
        this.cep = cep
    }

    override fun toString(): String {
        return """Endereco{logradouro='$logradouro${'\''.toString()}, numero='$numero${'\''.toString()}, complemento='$complemento${'\''.toString()}, bairro='$bairro${'\''.toString()}, cidade='$cidade${'\''.toString()}, uf='$uf${'\''.toString()}, cep='$cep${'\''.toString()}${'}'.toString()}"""
    }

}